pandas/core/reshape/concat.py

```diff
@@ -351,13 +351,16 @@ def concat(
     >>> df7
         a   b
     0   1   2
-    >>> new_row = pd.Series([3])
-    >>> new_row
-    0   3
-    >>> pd.concat([df7, new_row.to_frame().T], ignore_index=True)
-         a    b    0
-    0  1.0  2.0  NaN
-    1  NaN  NaN  3.0
+    >>> new_rows = pd.Series({'a': 3, 'b': 4})
```

Comment:
this will appear as a single row in the DataFrame, so `new_row` should be fine as a name

Reply:
Fixed in 95139122722c3827cf34a6db7862ac0eeffaf727